frontend/src/components/Post/PostPicture.js

```diff
@@ -1,56 +1,47 @@
 import React from "react";
+import { useState } from "react";
 import axios from "axios";
-import '../../config';
+import "../../config";
 
-class PostPicture extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            errorMsg: ""
-        }
-    }
-    submit(event) {
-        event.preventDefault();
-        const data = new FormData();
-        data.append('picture', event.target.picture.files[0]);
-        data.append('caption', event.target.caption.value);
-        axios.post(global.config.BACKEND_URL + "/picture", data)
-            .then(
-                (response) => {
-                    this.props.history.push(`/post/${response.data.id}`);
-                },
-                (e) => {
-                    if (e.response) {
-                        if(e.response.status === 401) {
-                            this.setState({errorMsg: "You must be logged in."});
-                        } else {
-                            this.setState({errorMsg: e.response.data.message});
-                        }
-                    } else {
-                        this.setState({errorMsg: "An unknown error occured."});
-                    }
-                }
-            )
-    }
+export const PostPicture = props => {
```

Comment:
you can remove the `export` since you already declare the export on line 47

Reply:
Actually I think we are moving to a different format: it should be `export default function PostPicture(props) {....}` and then remove line 47 altogether